### PR TITLE
Fix: Missing Player transform

### PR DESCRIPTION
**Pull request by mikhail-dcl**
- Fix a CRDT exception happening at `145,60` and `146,59`. It fixes the reason why they were not started (not updated) at all
   - There is still an issue with `signedFetch` 😢 #1236 
- Unify Realm launch options:
   - moved all the related settings to a single class (Static position and initial parcel was in SO)
   - wrote the single Editor for it

### CODE REVIEW

File: Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs [hunk 1]
```diff
@@ -49,9 +50,17 @@ public SDKComponentCommandBufferSynchronizer(IComponentPool<T> componentPool)
...
-                        pointerToPrevObj = c;
+                        try
+                        {
+                            ref T pointerToPrevObj = ref world.Get<T>(entity);
+                            componentPool.Release(pointerToPrevObj);
+                            pointerToPrevObj = c;
+                        }
+                        catch (Exception)
+                        {
+                            {}
```
Comment (lorux0) [inline Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs:61]: Can be deleted

File: Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs [hunk 2]
```diff
@@ -49,9 +50,17 @@ public SDKComponentCommandBufferSynchronizer(IComponentPool<T> componentPool)
                     }
                     else
                     {
-                        ref T pointerToPrevObj = ref world.Get<T>(entity);
-                        componentPool.Release(pointerToPrevObj);
-                        pointerToPrevObj = c;
+                        try
```
Comment (lorux0) [inline Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs:53]: What is the reason of handling the exception here? Using TryGetRef could be useful in this case?:
```
ref T pointerToPrevObj = ref world.TryGetRef<T>(entity, out bool hasComponent);
```

File: Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs [hunk 3]
```diff
@@ -34,18 +34,20 @@ namespace Global.Dynamic
...
-
-        [SerializeField]  [ShowIfEnum("initialRealm", (int)InitialRealm.Localhost)]
-        private string remoteSceneID = "bafkreihpuayzjkiiluobvq5lxnvhrjnsl24n4xtrtauhu5cf2bk6sthv5q";
-
-        [SerializeField]  [ShowIfEnum("initialRealm", (int)InitialRealm.Localhost)]
-        private ContentServer remoteSceneContentServer = ContentServer.World;
-        
+        [SerializeField] private RealmLaunchSettings launchSettings;
+
+        // [SerializeField] private InitialRealm initialRealm;
```
Comment (lorux0) [inline Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs:39]: Can be deleted

**Review by lorux0 — APPROVED** [review]

File: Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs (hunk 2 above)
Comment (mikhail-dcl) [reply Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs]: no reason, debug stuff

File: Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs [hunk 4]
```diff
@@ -4,6 +4,7 @@
 using DCL.Optimization.Pools;
 using ECS.LifeCycle.Components;
 using JetBrains.Annotations;
+using System;
```
Comment (popuz) [inline Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs:7]: probably it is some leftover and can be removed

**Review by popuz — APPROVED** [review]
Some CRDT magic 🪄 LGTM 🟢

**Pull request merged**